lms/static/js/ccx/schedule.js

```diff
@@ -460,24 +467,25 @@ var edx = edx || {};
           var date = $(this).find('input[name=date]').val(),
           time = $(this).find('input[name=time]').val();
           var valid_date = new Date(date);
-          if (isNaN(valid_date.valueOf())) {
+          var dateTimeIsNull = (!date && !time) ? true: false;
+          if (isNaN(valid_date.valueOf()) && !dateTimeIsNull) {
```

Comment:
Can we just write
```javascript
if(!dateTimeIsNull){
   all other stuff here
}
```
instead of checking in each case?

Reply:
Good point, but I only change until this part 
https://github.com/Livit/Livit.Learn.EdX/pull/110/files#diff-ec64fd1a163a7a55176fae8c4d71b333R482

because I think it's simpler, CMIIW